rapids-cmake/cuda/set_architectures.cmake

```
@@ -59,11 +59,10 @@ function(rapids_cuda_set_architectures mode)
   if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA")
 
     if(CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL 13.0.0)
-      set(supported_archs "75-real" "80-real" "86-real" "90a-real" "100f-real" "120a-real"
-                          "121f-real" "121-virtual")
+      set(supported_archs "75-real" "80-real" "86-real" "90a-real" "100fg-real" "120a-real" "120")
```

Unless `fg` is yet another new thing that I don't know about...
```suggestion
      set(supported_archs "75-real" "80-real" "86-real" "90a-real" "100f-real" "120a-real" "120")
```